__tests__/deleteUsers.test.js

```diff
@@ -21,8 +21,8 @@ const buildPrepareDeleteRequestTestCases = () => {
   .reduce((acc, currentVal) => {
     try {
       const delUsers = require(`../${version}/destinations/${currentVal}/deleteUsers`);
-      const inputJson = require(`./data/${currentVal}_deleteUsers_req_input.json`);
-      const outputJson = require(`./data/${currentVal}_deleteUsers_req_output.json`);
+      const inputJson = require(`./data/deleteUsers/${currentVal}/prepare_req_input.json`);
+      const outputJson = require(`./data/deleteUsers/${currentVal}/prepare_req_output.json`);
```

Comment:
With the `validateMockAxiosClientReqParams`, do we still need separate req preparation tests?

Reply:
The only reason might help us in catching some corner cases wrt how the request is being prepared. But if we feel they're redundant, we can remove them as well

Reply:
Let's remove them, as those corner cases also can be tested well now.

Reply:
Removed them